Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -90,16 +92,24 @@
     trackingGeometry,
     field,
     CKFPerformanceConfig(ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0, nMeasurementsMin=6),
+    TrackSelectorRanges(pt=(1.0 * u.GeV, None), absEta=(None, 4.0), removeNeutral=True),
+    outputDirRoot=outputDir,
+)
+
+addAmbiguityResolution(
+    s,
+    AmbiguityResolutionConfig(
+        maximumSharedHits=3,
+    ),
```

Comment:
```suggestion
    AmbiguityResolutionConfig(maximumSharedHits=3),
```
This seems to be overzealous `black` formatting. I've noticed how it can put things on separate lines, but then doesn't go back when the option is simplified - possibly because of the extra `,` it adds.

Reply:
oh good point will do the same for odd